Description: fix paths since our build differs from upstream one
Author: Xavier Guimard <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2021-10-01

--- a/defer-to-connect/package.json
+++ b/defer-to-connect/package.json
@@ -2,9 +2,9 @@
   "name": "defer-to-connect",
   "version": "2.0.1",
   "description": "The safe way to handle the `connect` socket event",
-  "main": "dist/source",
+  "main": "dist",
   "files": [
-    "dist/source"
+    "dist"
   ],
   "engines": {
     "node": ">=10"
@@ -54,7 +54,7 @@
   },
   "nyc": {
     "include": [
-      "dist/source"
+      "dist"
     ],
     "extension": [
       ".ts"
@@ -67,5 +67,5 @@
       }
     }
   },
-  "types": "dist/source/index.d.ts"
+  "types": "dist/index.d.ts"
 }
--- a/package.json
+++ b/package.json
@@ -5,7 +5,7 @@
 	"license": "MIT",
 	"repository": "sindresorhus/got",
 	"funding": "https://github.com/sindresorhus/got?sponsor=1",
-	"main": "dist/source",
+	"main": "dist",
 	"engines": {
 		"node": ">=10.19.0"
 	},
@@ -16,7 +16,7 @@
 		"prepare": "npm run build"
 	},
 	"files": [
-		"dist/source"
+		"dist"
 	],
 	"keywords": [
 		"http",
@@ -93,7 +93,7 @@
 		"typescript": "4.0.3",
 		"xo": "^0.34.1"
 	},
-	"types": "dist/source",
+	"types": "dist",
 	"sideEffects": false,
 	"ava": {
 		"files": [
--- a/szmarczak-http-timer/package.json
+++ b/szmarczak-http-timer/package.json
@@ -2,7 +2,7 @@
 	"name": "@szmarczak/http-timer",
 	"version": "4.0.6",
 	"description": "Timings for HTTP requests",
-	"main": "dist/source",
+	"main": "dist",
 	"engines": {
 		"node": ">=10"
 	},
@@ -13,7 +13,7 @@
 		"coveralls": "nyc report --reporter=text-lcov | coveralls"
 	},
 	"files": [
-		"dist/source"
+		"dist"
 	],
 	"keywords": [
 		"http",
@@ -47,7 +47,7 @@
 		"typescript": "^4.3.5",
 		"xo": "^0.39.1"
 	},
-	"types": "dist/source",
+	"types": "dist",
 	"nyc": {
 		"extension": [
 			".ts"
